docs/packaging-an-application/template-functions.md

```diff
@@ -663,24 +663,6 @@ PremkitNetworkName() string
 
 PremkitNetworkName returns the name of the premkit docker network.
 
-```yml
-services:
-  voting-app:
-    image: gaiadocker/example-voting-app-vote:good
-    environment:
-      REPLICATED_INTEGRATIONAPI: {{repl PremkitAPIAddress }}
```

Comment:
Vendor will still need to add environment variables to access the Integration API, should we really remove this part as well?